doc/content/integrations/cloud-integrations/blockbax/_index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Blockbax"
+description: ""
+weight:
+aliases: ["/integrations/blockbax"]
+---
+
+[Blockbax](https://blockbax.com/) is a fully configurable and scalable cloud IoT platform which requires no programming.
+
+<!--more-->
+
+Check the [Blockbax documentation page](https://blockbax.com/docs/) for more info.
+
+## Prerequisites
+
+1. Own a Blockbax project or [request one](https://blockbax.com/about#contact) if you do not have an account yet.
+
+## Blockbax Setup
+
+[Login](https://login.blockbax.com/) to the Blockbax Platform.
+
+Create a [subject type](https://blockbax.com/docs/subjects/#managing-subject-types) and configure metrics. External IDs for these metrics  need to match the property names returned by the uplink payload formatter that will be [configured on {{% tts %}}]({{< ref "/integrations/cloud-integrations/blockbax#configure-the-things-stack" >}}).
+
+{{< figure src="blockbax-metric-external-id-in-payload.png" alt="Relating Blockbax metrics to the Things Stack" >}}
+
+Additionally, you can choose to [create subjects](https://blockbax.com/docs/subjects/#creating-subjects) manually with external IDs matching the device IDs or set the `autoCreateSubjects` option in {{% tts %}} payload formatter function to `true`.
```

Comment:
This part is a bit unclear. What is the difference between creating subjects manually and creating a subject type? When you say that manually created subjects need to have external IDs matched to device IDs, do you mean TTS device IDs? And what does the `autoCreateSubjects` option do? Can you elaborate that? Feel free to add another commit to explain this a bit for a reader.

Reply:
Hi @nejraselimovic, thanks for reviewing our contribution - good feedback! 

In a nutshell a subject type is a blueprint you will always need to create manually. A subject is an instance of this type and can be created for you. For example you need to create the generic model of a building (= subject type), but then you can easily add new buildings (= subjects) manually or use `autoCreateSubjects`.

I added some more generic description about the Blockbax concepts and elaborated on `autoCreateSubjects`. Do you think this clears things up?